tests/aws_iot_secure_tunneling_client_test.c

```diff
@@ -135,6 +145,24 @@ int main(int argc, char **argv) {
     ASSERT_SUCCESS(aws_condition_variable_wait(&condition_variable, &mutex));
     aws_mutex_unlock(&mutex);
 
+    if (local_proxy_mode == AWS_SECURE_TUNNELING_SOURCE_MODE) {
+        AWS_RETURN_ERROR_IF2(aws_secure_tunnel_stream_start(secure_tunnel) == AWS_OP_SUCCESS, AWS_OP_ERR);
+        ASSERT_SUCCESS(aws_condition_variable_wait(&condition_variable, &mutex));
+
+        char *payload = "Hi! I'm Paul / Some random payload\n";
+        struct aws_byte_buf buffer = aws_byte_buf_from_c_str(payload);
+        struct aws_byte_cursor cur = aws_byte_cursor_from_buf(&buffer);
```

Comment:
Cursors can be created directly from static C strings: https://github.com/awslabs/aws-c-common/blob/master/include/aws/common/byte_buf.h#L61

Reply:
Fixed it.